### DISCUSSION TITLE
About -fno-crossjumping -fno-gcse

### DISCUSSION THREAD

**Discussion by esrrhs** [edubart/nelua-lang#103] [General]
I test the two option, and the benchmark result is nearly same. here is the code
```
function fibonacci(n)
    if n<3 then
        return 1
    else
        return fibonacci(n-1) + fibonacci(n-2)
    end
end

print(fibonacci(39))
```

**Comment by edubart** [comment]
This example is too simple for benchmark what `-fno-crossjumping -fno-gcse` can do, you can read more about this flag and Lua interpreter optimization in general in this thread:
http://lua-users.org/lists/lua-l/2020-08/msg00153.html

At some point Dibyendu Majumdar told about using `-fno-crossjumping -fno-gcse` in the Lua VM to speedup and I got 4% speedup of the Nelua compiler, which is much complex than that.
http://lua-users.org/lists/lua-l/2020-08/msg00165.html
